src/twitter/Timeline.js

```diff
@@ -426,11 +426,15 @@ class Timeline {
     for (const room in memberLists) {
       if (memberLists[room].realJoinedUsers.length === 0) {
         log.silly("Timeline", "%s has no real users", room);
+        // Room is empty, lets ignore it
         this._empty_rooms.add(room);
       } else {
+        // Room is not empty, delete if it exists
         this._empty_rooms.delete(room);
+        // Store the number of bot users away
         this._members_rooms.set(room.id, memberLists[room].remoteJoinedUsers);
         log.verbose("Timeline", "Clearing hot_room_counter");
+        // Clear the collection of hot rooms and let them reset to 0.
         this._hot_room_counter.clear();
```

Comment:
I'm not sure the counter will be that accurate if it is not read immediately before being cleared.